task/shp/0.1/shp.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: shp
+  labels:
+    app.kubernetes.io/version: "1.20"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.0"
+    tekton.dev/categories: CLI
+    tekton.dev/tags: cli
+    tekton.dev/displayName: "Shipwright CLI"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  workspaces:
+    - name: kubeconfig
+      description: >-
+        An optional workspace that allows you to provide a .kube/config
+        file for shp to access the cluster. The file should be placed at
+        the root of the Workspace with name kubeconfig.
+      optional: true
+  description: This task performs operations on Shipwright resources using shp
+  params:
+    - name: SHP_IMAGE
+      description: shp CLI container image to run this task
+      default: quay.io/siamaksade/shipwright-cli:0.12
```

Comment:
hey @siamaksade 👋🏻, is there any public image available for this?

Reply:
Hey. updated to the cli image on shipwright quay.io org